docs/develop/api-reference/auth-tokens.md

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 7
+title: Authentication control token API reference information
+sidebar_label: Authentication control tokens
+description: Learn the authentication control token API calls you need to know about and how to use them with Momento services.
+---
+
+import { SdkExampleTabs } from "@site/src/components/SdkExampleTabs";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleTabs to SdkExampleTabsImpl
+import { SdkExampleTabsImpl } from "@site/src/components/SdkExampleTabsImpl";
+
+# Authentication control API reference
```

Comment:
Still conflicted about using `Authentication` here vs `Auth` to denote both AuthZ and AuthN

Reply:
Changed to auth.